src/helpers/validateSignUpEmailAddress.js

```diff
@@ -2,8 +2,8 @@ const REGEX = /^(?:(?:[^<>()[\]\\.,;:\s@"]+(?:\.[^<>()[\]\\.,;:\s@"]+)*)|(?:".+"
 
 const validateSignUpEmailAddress = (email, emailDomains) => {
     if(emailDomains === undefined) {
-        if (process.env.NEXT_PUBLIC_SIGN_UP_EMAIL_DOMAIN === undefined)
-            return true;
+        // if (process.env.NEXT_PUBLIC_SIGN_UP_EMAIL_DOMAIN === undefined)
```

Comment:
This will almost certainly throw an error if it is ever undefined.
Not sure what the purpose of this edit is.